lib/scaffold/start.js

```diff
@@ -187,7 +194,44 @@ function start(options) {
 
 }
 
+/**
+ * This function will fork the passed in process and exit the parent process
+ * in order to daemonize the process. If there is already a daemon for this pid (process),
+ * then the function just returns. Stdout and stderr both append to one file, 'bitcore-node.log'
+ * located in the datadir.
+ * @param {String} datadir - The data directory where the bitcoin blockchain and config live.
+ * @param {Object} _process - The process that needs to fork a child and then, itself, exit.
+ */
+function spawnChildProcess(datadir, _process) {
+
+  if (_process.env.__bitcore_node) {
```

Comment:
yes, this can be called twice since it can still be called from the forked process (which is already daemonized). It is best to check if the passed in process has already been daemonized.